desktop/app/menu-helper.js

```diff
@@ -1,5 +1,5 @@
 // Can't tell which thread we're in so let's try both
-const electron = require('electron')
+import electron from 'electron'
```

Comment:
let's avoid mixing imports and requires from now on. If you absolutely need a require() you have to use require().default else you sometimes won't get the default exported var